k8s/tests/check-network.sh

```diff
@@ -12,36 +12,15 @@ check_status() {
   status | jq -r 'if (.status != "Failure") and (.status.readyReplicas == .status.replicas) then true else false end'
 }
 
-spec_peers() {
-  status | jq -r '.status.replicas'
-}
-
-available_peers() {
```

Comment:
Does your test have a CAS node? The issue I had was `if [ "$(available_peers)" == "$(spec_peers)" ];`  fails since `available_peers = spec_peers + 1` since CAS is present in the peers.json file.

Reply:
Ohh huh. Yes, it does have a CAS node.

The last test I ran was without checking for an exact number of peers though. I just checked if the number of available peers was 0.
```
available_peers() {
  jq -r '. | length' < /peers/peers.json
}

populate_peers() {
  if [ "$(available_peers)" == "0" ]; then
    exit 1
  fi

  mkdir /config/env
  .
  .
```

Reply:
Even before that though, I was using the default config and it was working in GCP for me with the local CAS...